include/event2/http.h

```diff
@@ -571,7 +571,7 @@ enum evhttp_cmd_type {
 	EVHTTP_REQ_MOVE    = 1 << 15,
 };
 
-#define EVHTTP_REQ_MAX EVHTTP_REQ_PATCH
+#define EVHTTP_REQ_MAX EVHTTP_REQ_MOVE
```

Comment:
This should be fixed in a separate patch (the original `EVHTTP_REQ_UNKNOWN_` const)

As for exporting this in a public header this has pros and cons:
- pros: can be used in user code
- cons: will be used in user code and hence can break programs after new methods will be added (although I guess that users will stick to the last method in this define anyway, so not sure which one I prefer)

Reply:
I don't see how it is related to  `EVHTTP_REQ_UNKNOWN_` ?
user should use values > `EVHTTP_REQ_MAX ` and < `EVHTTP_REQ_UNKNOWN_`

Reply:
Sorry, my mistake, didn't look carefully